eraser-map.yaml

```diff
@@ -1140,8 +1138,8 @@ layers:
                     color: [[8, *highway1], [14, *highway1], [15, [1.0,1.0,1.0]], [17, [1.0,1.0,1.0]]]
                     width: [[8, 1px], [9, 1.25px], [11, 1.2px], [12, 1.3px], [13, 1.5px], [14, 2.25px], [15, 0.15px], [16, 5.5px], [17, 12m]]
                     outline:
-                        color: [[9, *highway_casing1], [13, *highway_casing1], [15,[0.988,0.495,0.439]], [17, [0.988,0.495,0.439]]]
-                        width: [[9, 0.5px], [12, 1px], [15, 2px], [16, 2.25px], [17, 3px], [18, 3px]]
+                        color: [[13, *highway_casing1], [15,[0.988,0.495,0.439]], [17, [0.988,0.495,0.439]]]
+                        width: [[12, 0], [13, 1.33px], [15, 2px], [16, 2.25px], [17, 3px], [18, 3px]]
```

Comment:
I think that's fine since highways at z12 are not so high-density compared to outline rules affecting all roads.